qemu/tests/win_heavyload.py

```diff
@@ -122,26 +122,27 @@ def add_option(cmd, key, val):
         logging.info("heavyload cmd: %s" % start_cmd)
         session.sendline(start_cmd)
         if not loop_session_cmd(session, check_running_cmd):
-            raise error.TestError("heavyload process is not started")
+            test.error("heavyload process is not started")
 
         sleep_before_migration = int(params.get("sleep_before_migration",
                                                 "0"))
         time.sleep(sleep_before_migration)
 
-        error.context("Verify vm is alive", logging.info)
+        error_context.context("Verify vm is alive", logging.info)
         utils_misc.wait_for(vm.verify_alive,
-                            timeout=test_timeout, step=steping)
+                            timeout=test_timeout*60, step=steping)
     finally:
         # in migration test, no need to stop heavyload on src host
         cleanup_in_the_end = params.get("unload_stress_in_the_end", "yes")
         if cleanup_in_the_end == "yes":
-            error.context("Stop load and clean tmp files", logging.info)
+            error_context.context("Stop load and clean tmp files", logging.info)
             if not installed and download_url:
                 utils.system("rm -f %s/HeavyLoad*.exe" % tmp_dir)
                 session.cmd("del /f /s %sHeavyLoad*.exe" % dst)
-            if loop_session_cmd(session, check_running_cmd):
-                if not loop_session_cmd(session, stop_cmd):
-                    raise error.TestFail("Unable to terminate heavyload "
-                                         "process")
+            if not session.cmd_status(check_running_cmd):
+                try:
+                    session.cmd(stop_cmd, timeout=120)
```

Comment:
Why only try once now ?

Reply:
@huangyum  It is only a clear step, warning is enough. and i use "timeout" replace the repetitive operation.